sample-compose/app/build.gradle.kts

```diff
@@ -141,17 +152,18 @@ dependencies {
     debugImplementation("com.github.chuckerteam.chucker:library:${Versions.CHUCKER_VERSION}")
     releaseImplementation("com.github.chuckerteam.chucker:library-no-op:${Versions.CHUCKER_VERSION}")
 
-    // Testing
+    // Unit test
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.KOTLINX_COROUTINES_VERSION}")
     testImplementation("io.kotest:kotest-assertions-core:${Versions.TEST_KOTEST_VERSION}")
     testImplementation("junit:junit:${Versions.TEST_JUNIT_VERSION}")
-    testImplementation("androidx.test:core:${Versions.TEST_ANDROIDX_CORE_VERSION}")
-    testImplementation("androidx.test:runner:${Versions.TEST_RUNNER_VERSION}")
-    testImplementation("androidx.test:rules:${Versions.TEST_RUNNER_VERSION}")
-    testImplementation("androidx.test.ext:junit-ktx:${Versions.TEST_JUNIT_ANDROIDX_EXT_VERSION}")
-    testImplementation("com.google.dagger:hilt-android-testing:${Versions.HILT_VERSION}")
-    testImplementation("org.jetbrains.kotlin:kotlin-reflect:${Versions.KOTLIN_REFLECT_VERSION}")
     testImplementation("io.mockk:mockk:${Versions.TEST_MOCKK_VERSION}")
-
-    kaptTest("com.google.dagger:hilt-android-compiler:${Versions.HILT_VERSION}")
-    testAnnotationProcessor("com.google.dagger:hilt-android-compiler:${Versions.HILT_VERSION}")
+    testImplementation("app.cash.turbine:turbine:${Versions.TEST_TURBINE}")
+
+    // Instrument test
+    // Need to have BOM for androidTestImplementation https://github.com/gradle/gradle/issues/23347
+    androidTestImplementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
+    androidTestImplementation("androidx.test:rules:${Versions.TEST_RULES_VERSION}")
+    androidTestImplementation("io.mockk:mockk-android:${Versions.TEST_MOCKK_VERSION}")
+    androidTestImplementation("io.mockk:mockk-agent-android:${Versions.TEST_MOCKK_VERSION}")
```

Comment:
Interesting, first time I see this dependency. What's the purpose of this one? 🙏

Reply:
@Tuubz Alright, we don't need this when running UI tests on device API level >= 28 (Android P). We don't support to run UI test before P this time, the support should be done with `dexopener` dependency https://mockk.io/ANDROID.html.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/bda2a6a742ae459b455588e700ca4d387716ec29